autoload/fern/internal/drawer/hover_popup.vim

```diff
@@ -15,16 +15,24 @@ function! fern#internal#drawer#hover_popup#init() abort
 
   augroup fern_internal_drawer_hover_popup_init
     autocmd! * <buffer>
-    autocmd CursorMoved <buffer> call timer_stop(s:timer) | call s:hide() | let s:timer = timer_start(g:fern#drawer_hover_popup_delay, { -> s:show(fern#helper#new()) })
-    autocmd BufLeave <buffer> call timer_stop(s:timer) | call s:hide()
+    autocmd CursorMoved <buffer> call s:debounced_show()
+    autocmd BufLeave <buffer> call s:hide()
   augroup END
 endfunction
 
 function! s:available() abort
-  return has('nvim') || exists('*popup_create')
+  let has_win = has('nvim') || exists('*popup_create')
+  return has_win && exists('*win_execute')
+endfunction
+
+function! s:debounced_show() abort
+  call s:hide()
+  let s:show_timer = timer_start(g:fern#drawer_hover_popup_delay, { -> s:show(fern#helper#new())  })
```

Comment:
NOTE:

Initialization of the `helper` instance will be changed in the next commit.